internal/service/alarms/internal/db/migrations/000006_create_alarm_service_configuration_table.down.sql

```diff
@@ -0,0 +1,8 @@
+-- Drop the trigger that updates updated_at for alarm_service_configuration
```

Comment:
I've had this discussion with @pixelsoccupied before... IMO, we shouldn't be adding any down commands for this set of initial scripts.  These initial scripts essentially represent our "baseline" schema and later once we've reached a more productized state we can start adding incremental updates with related down scripts if necessary.

Reply:
ok